lib/Dialect/AIE/Transforms/AIECreatePathFindFlows.cpp

```diff
@@ -125,13 +125,10 @@ struct ConvertFlowsToInterconnect : OpConversionPattern<FlowOp> {
                           srcBundle, srcChannel, WireBundle::North, shimCh);
           } else if (srcBundle ==
                      WireBundle::PLIO) { // PLIO at start of flows with mux
-            if (srcChannel == 2 || srcChannel == 3 || srcChannel == 6 ||
-                srcChannel == 7) { // Only some PLIO requrie mux
-              ShimMuxOp shimMuxOp = analyzer.getShimMux(rewriter, srcSB.col);
-              addConnection(
-                  rewriter, cast<Interconnect>(shimMuxOp.getOperation()),
-                  flowOp, srcBundle, srcChannel, WireBundle::North, shimCh);
-            }
+            ShimMuxOp shimMuxOp = analyzer.getShimMux(rewriter, srcSB.col);
```

Comment:
It's unclear to me what the point of this change is, but I don't understand the original code necessarily.  Is this related to the fact that you only 'support' channel 0 and 1 in the code below?

Reply:
I don't know how correct the previous code was, but in the pass here: https://github.com/Xilinx/mlir-aie/blob/d08090b51dc480ea1eacb9d16d8770a463093047/lib/Targets/AIETargetXAIEV2.cpp#L709-L767 we use the information in the shim mux to know whether we should enable PLIO or GMIO.